libs/dh/market-participant/actors/feature-actors/src/lib/create/dh-actors-create-actor-modal.component.ts

```diff
@@ -120,8 +172,19 @@ export class DhActorsCreateActorModalComponent {
     });
   }
 
+  onOrganizationChange(id: string): void {
+    this._getOrginationByIdQuery$.setVariables({ id });
+    this._getOrginationByIdQuery$.valueChanges
+      .pipe(takeUntilDestroyed(this._destroyRef))
+      .subscribe((result) => {
```

Comment:
issue: this creates a new subscription every time an org is selected. It's better to reuse the same subscription, and only set the query variable.